_assets/ci/Jenkinsfile.desktop

```
@@ -0,0 +1,135 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@s5cmd-windows'
+
+pipeline {
+  /* This way we run the same Jenkinsfile on different platforms. */
+  agent { label "${params.AGENT_LABEL}" }
+
+  parameters {
+    string(
+      name: 'BRANCH',
+      defaultValue: 'ci/develop',
+      description: 'Name of branch to build.'
+    )
+    string(
+      name: 'AGENT_LABEL',
+      description: 'Label for targetted CI slave host.',
+      defaultValue: params.AGENT_LABEL ?: getAgentLabel(),
+    )
+    booleanParam(
+      name: 'RELEASE',
+      defaultValue: false,
+      description: 'Enable to create build for release.',
+    )
+  }
+
+  options {
+    timestamps()
+    ansiColor('xterm')
+    /* Prevent Jenkins jobs from running forever */
+    timeout(time: 10, unit: 'MINUTES')
+    disableConcurrentBuilds()
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '5',
+      daysToKeepStr: '30',
+      artifactNumToKeepStr: '1',
+    ))
+  }
+
+  environment {
+    PLATFORM = getPlatformFromLabel(params.AGENT_LABEL)
+    TMPDIR   = "${WORKSPACE_TMP}"
+    GOPATH   = "${WORKSPACE_TMP}/go"
+    GOCACHE  = "${WORKSPACE_TMP}/gocache"
+    PATH     = "${PATH}:${GOPATH}/bin"
+    REPO_SRC = "${GOPATH}/src/github.com/status-im/status-go"
+    VERSION  = sh(script: "./_assets/scripts/version.sh", returnStdout: true)
+    ARTIFACT = utils.pkgFilename(
+      name:    'status-go',
+      type:    env.PLATFORM,
+      version: env.VERSION,
+      ext:     'zip',
+    )
+  }
+
+  stages {
+    stage('Setup') {
+        steps {
+          script {
+            if (env.PLATFORM != 'windows') {
+                sh "mkdir -p \$(dirname ${REPO_SRC})"
+                sh "ln -s ${WORKSPACE} ${REPO_SRC}"
```

To help with your case of existing file error 
```suggestion
                sh "ln -nsf ${WORKSPACE} ${REPO_SRC}"
```
ref -> https://www.man7.org/linux/man-pages/man1/ln.1.html

---

No need for a `force` flag because this error is just from me testing the changes in nix setup and since it failed, `make deep-clean` couldn't clean the workspace so no worries

---

> just from me testing the changes

Oh thanks for explaining.

---

I didn't check, The PR is still in Draft mode :)